### PR TITLE
Changes to DB

### PR DESCRIPTION
**Pull request by RobertoAldea**
Added Power and PwrMode to generic measures, added disable to PwrMode, added updating data to cache db

### CODE REVIEW

File: code/drv_db/src/wattrex_driver_db/drv_db_dao_base.py [hunk 1]
```diff
@@ -59,6 +59,9 @@ class DrvDbBaseGenericMeasureC(Base):
     Timestamp = Column(DateTime, nullable=False)
     Voltage = Column(MEDIUMINT(), nullable=False)
     Current = Column(MEDIUMINT(), nullable=False)
+    Power = Column(MEDIUMINT())
+    PwrMode = Column(Enum(DrvDbCyclingModeE))
```
Comment (mariuscrsn) [inline code/drv_db/src/wattrex_driver_db/drv_db_dao_base.py]: Use get_all_values

File: code/drv_db/src/wattrex_driver_db/drv_db_dao_master.py [hunk 2]
```diff
@@ -195,30 +210,30 @@ class DrvDbInstructionC(Base):
...
                     {'extend_existing': True},)
 
-    CSID = Column(ForeignKey('CyclerStation.CSID'), nullable=False)
-    BatID = Column(ForeignKey('Battery.BatID'), nullable=False)
-    ProfID = Column(ForeignKey('Profile.ProfID'), nullable=False)
+    CSID = Column(ForeignKey(DrvDbCyclerStationC.CSID), nullable=False)
+    BatID = Column(ForeignKey(DrvDbBatteryC.BatID), nullable=False)
+    ProfID = Column(ForeignKey(DrvDbProfileC.ProfID), nullable=False)
 
     # CyclerStation = relationship('DrvDbCyclerStationC')
```
Comment (mariuscrsn) [inline code/drv_db/src/wattrex_driver_db/drv_db_dao_master.py]: Remove comments

**Review by mariuscrsn — CHANGES_REQUESTED** [review]

File: code/drv_db/src/wattrex_driver_db/drv_db_dao_base.py (hunk 1 above)
Comment (RobertoAldea) [reply code/drv_db/src/wattrex_driver_db/drv_db_dao_base.py]: Done

File: code/drv_db/src/wattrex_driver_db/drv_db_dao_master.py (hunk 2 above)
Comment (RobertoAldea) [reply code/drv_db/src/wattrex_driver_db/drv_db_dao_master.py]: Done

**Review by mariuscrsn — APPROVED** [review]

**Pull request merged**